packages/amplify-category-api/package.json

```diff
@@ -18,7 +18,11 @@
     "inquirer": "^6.0.0",
     "moment": "^2.22.2",
     "opn": "^5.3.0",
-    "uuid": "^2.0.3"
+    "uuid": "^2.0.3",
+    "aws-sdk": "^2.391.0",
+    "ora": "^3.0.0",
+    "graphql": "^0.13.2",
+    "graphql-relational-schema-transformer": "../graphql-relational-schema-transformer"
```

Comment:
Hey, you shouldn't be referring to the packages in a monrepo like this. Take a look at other packages imported by the transformer.

Reply:
done - switched to normal package referencing